bin/gobuild.sh

```diff
@@ -63,6 +63,14 @@ while read -r line; do
     LD_EXTRAFLAGS="${LD_EXTRAFLAGS} -X ${line}"
 done < "${BUILDINFO}"
 
+# verify go version
+minimum_go_version=go1.13
+IFS=" " read -ra go_version <<< "$(${GOBINARY} version)"
```

Comment:
Can add comments about which patterns are we trying to allow ?

Reply:
Ref: https://github.com/kubernetes/kubernetes/blob/master/hack/lib/golang.sh#L470

Reply:
Please add a comment in the script that says this was copied verbatim from Kubernetes hack.
Also add a comment about what patterns this is looking for.
Thanks

Reply:
I would prefer to make this a warning. If a user has older go, it will fail currently, so warning vs failing is essentially the same. But if we get a false positive here, then we are just failing the build for no reason.

Who knows what kind of responses `go version` may give.. I would guess not ever build of go reports the version like this

Reply:
also in `go.mod` there is a version, will that give a warning/error if it is above the current version?